include/braincloud/BrainCloudCustomEntity.h

```diff
@@ -208,8 +208,8 @@ namespace BrainCloud {
 		 */
 		void incrementData(const char * in_entityType, const char * in_entityId, const std::string& in_fieldsJson, IServerCallback * in_callback = NULL);
 
-
+		
 	private:
 		BrainCloudClient * m_client;
 	};
-}
\ No newline at end of file
+}
```

Comment:
Dont commit that file. It's preferable to have the extra return at the end of the file, but you didn't actually have to touch that file for this work so dont commit this.